Makefile

```diff
@@ -36,10 +36,10 @@ Documentation/html/git-filter-repo.html:
 	git show docs:html/git-filter-repo.html >Documentation/html/git-filter-repo.html
 
 install: snag_docs #fixup_locale
-	cp -a git-filter-repo $(bindir)/
-	ln -s $(bindir)/git-filter-repo $(pythondir)/git_filter_repo.py
-	cp -a Documentation/man1/git-filter-repo.1 $(mandir)/man1/git-filter-repo.1
-	cp -a Documentation/html/git-filter-repo.html $(htmldir)/git-filter-repo.html
+	cp -a git-filter-repo "$(bindir)/"
+	ln -sf "$(bindir)/git-filter-repo" "$(pythondir)/git_filter_repo.py"
+	cp -a Documentation/man1/git-filter-repo.1 "$(mandir)/man1/git-filter-repo.1"
+	cp -a Documentation/html/git-filter-repo.html "$(htmldir)/git-filter-repo.html"
```

Comment:
Thanks for the contributions!  I really need to set up some guidelines for folks contributing (e.g. I hate GitHub PRs -- see https://public-inbox.org/git/CABPp-BG2SkH0GrRYpHLfp2Wey91ThwQoTgf9UmPa9f5Szn+v3Q@mail.gmail.com/); but since I haven't done that yet...

In general, I want to follow the same guidelines git.git uses (see https://git.kernel.org/pub/scm/git/git.git/tree/Documentation/SubmittingPatches?id=941790d7de).

The code changes in this patch look good, but the commit message needs some fixes:
  - prefix the summary with the area, and drop the trailing colon "Makefile: fix path installation issues"
  - the second line of the commit message should be blank
  - "wrap paths with parentheses" (you mean quotes) -> "quote paths that may have spaces"
  - Remove the "need to" from the second bullet
  - Add a Signed-off-by line as per the git.git contribution guidelines

Reply:
Hmm ok, how should I do that? Amend the changes locally, then force-push? or is that madness? I see in your email you mention rebasing being hard on the reviewer...
Sorry for the noodness, I'm not used to contributing (not that used to github either actually).

Reply:
Yeah, go ahead and fix each commit locally, then force-push.  GitHub PRs are kinda lame with rebases, though much less lame than they used to be, but it's more important to get history right.  Besides, these aren't that long of patches, so it's not a big deal anyway.